awx/ui/src/components/AppContainer/NavExpandableGroup.js

```diff
@@ -1,60 +1,47 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes, { oneOfType, string, arrayOf } from 'prop-types';
-import { matchPath, Link, withRouter } from 'react-router-dom';
+import { matchPath, Link, useHistory } from 'react-router-dom';
 import { NavExpandable, NavItem } from '@patternfly/react-core';
 
-class NavExpandableGroup extends Component {
-  constructor(props) {
-    super(props);
-    const { routes } = this.props;
+function NavExpandableGroup(props) {
+  const history = useHistory();
+  const { groupId, groupTitle, routes } = props;
 
-    // Extract a list of paths from the route params and store them for later. This creates
-    // an array of url paths associated with any NavItem component rendered by this component.
-    this.navItemPaths = routes.map(({ path }) => path);
-    this.isActiveGroup = this.isActiveGroup.bind(this);
-    this.isActivePath = this.isActivePath.bind(this);
-  }
+  // Extract a list of paths from the route params and store them for later. This creates
+  // an array of url paths associated with any NavItem component rendered by this component.
+  const navItemPaths = routes.map(({ path }) => path);
 
-  isActiveGroup() {
-    return this.navItemPaths.some(this.isActivePath);
+  function isActiveGroup() {
+    navItemPaths.some(isActivePath);
```

Comment:
This can just be a variable:
`const isActiveGroup = navItemPaths.some(isActivePath)`. And I'd probably rename it to `isActive` to match the prop it links with.

Reply:
Updated.